server/lomas_server/administration/mongodb_admin.py

```diff
@@ -9,6 +9,7 @@
 from pymongo.results import _WriteResult
 import yaml
 
+import pathmagic  # noqa
```

Comment:
this allows me to access the module admin_database outside the directory administration. Open to better solution, small hack i found on stackoverflow